modules/oc-mirror-imageset-config-params.adoc

```diff
@@ -269,3 +269,12 @@ channels:
 |Boolean. The default value is `false`.
 
 |===
+
+[IMPORTANT]
+====
+Sometimes error occurs when the minversion and maxversion are filtered out in the `ImageSetConfiguration` parameters. That leads to a failure in forming a directed acrylic graph from the remaining operator versions. Instead the previous graph is truncated and forms two or more separate graphs. 
+
+In the channel, after filtering, there are two or more paths that can be taken from the `minversion` to two different in `maxversion`. And in turn leads to the formation of `multiple channel heads`. 
+
+To avoid this error, do not filter out versions that will cause multiple channel heads or multiple update graphs to exist. Depending on the operator, that either the `maxversion` needs to be increased or the `minversion` needs to be decreased.
```

Comment:
```suggestion
To avoid this error, do not filter out the latest version of an operator. If you still run into the error, depending on the operator, either the `maxVersion` property needs to be increased or the `minVersion` property needs to be decreased. Because every operator graph can be different, you may need to adjust these values step-wise until the error is gone.
```